components/script/microtask.rs

```diff
@@ -71,6 +72,7 @@ impl MicrotaskQueue {
                             let _ = job.callback.Call_(&*target, ExceptionHandling::Report);
                         }
                     }
+                    Microtask::NotifyMutationObservers() => { }
```

Comment:
This still needs to invoke MutationObserver::notifyMutationObservers before anything will happen.